tds-dll-integration-tests/src/test/java/tds/student/dll/test/TestItemSelectionDLL.java

```diff
@@ -224,7 +224,7 @@ public void test_AA_GetNextItemCandidates_SP () throws Exception {
         }
       }
 
-      SingleDataResultSet sdrSet = _itemSelDLL.AA_GetNextItemCandidates_SP (_connection, oppkey);
+      SingleDataResultSet sdrSet = _itemSelDLL.AA_GetNextItemCandidates_SP (_connection, oppkey, true);
```

Comment:
Fixing some compile-time issues in the test submodule that look like they were due to interface changes we've made to the DLLs.